atmo/clusters/views.py

```diff
@@ -25,10 +31,10 @@ def new_cluster(request):
             )
         )
         return redirect('keys-new')
-    initial = {
-        'identifier': '{}-telemetry-analysis'.format(user_display(request.user)),
-        'size': 1,
-    }
+    elif ssh_key_count == 1:
+        # If only 1 ssh key, make it pre-selected.
+        initial['ssh_key'] = request.user.created_sshkeys.values('pk')[0]['pk']
```

Comment:
@robhudson Yeah, I wouldn't be concerned about speed of the cluster form view given the amount of traffic but would lean towards improving readability. But yeah, it probably doesn't matter. r+